erpnext/accounts/doctype/bank_transaction/bank_transaction.json

```
@@ -233,12 +236,30 @@
   {
    "fieldname": "column_break_oufv",
    "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "extended_bank_statement_section",
+   "fieldtype": "Section Break",
+   "label": "Extended Bank Statement"
+  },
+  {
+   "fieldname": "included_fee",
+   "fieldtype": "Currency",
+   "label": "Included Fee",
+   "options": "currency"
+  },
+  {
+   "description": "On save, the fees will be included in the deposit and withdraw amounts, as well as in the included fee.",
```

```suggestion
   "description": "On save, the Excluded Fee will be converted to an Included Fee.",
```

I feel like this is easier to understand